issues/models.py

```diff
@@ -34,15 +34,23 @@ def get_issue_area(value: str) -> str:
 class IssueManager(models.Manager):
     @transaction.atomic
     def set_area_issues_for_user(self, user: JustfixUser, area: str, issues: List[str]):
-        issues = list(set(issues))  # Remove duplicates.
-        self.filter(user=user, area=area).delete()
-        models = [
+        issues_set = set(issues)  # Remove duplicates and make lookup fast.
+        curr_models = list(self.filter(user=user, area=area))
+        models_to_delete = [model for model in curr_models if model.value not in issues_set]
+        for model in models_to_delete:
+            model.delete()
+        values_that_already_exist = set(
+            model.value
+            for model in curr_models if model.value in issues_set
+        )
+        models_to_create = [
             Issue(user=user, area=area, value=value)
-            for value in issues
+            for value in issues_set
+            if value not in values_that_already_exist
         ]
-        for model in models:
+        for model in models_to_create:
             model.full_clean()
-        self.bulk_create(models)
```

Comment:
Avoid bulk creation (and deletion) for now, to make it easy for us to add model-level change tracking behavior later, whether custom or via something like [django-reversion](https://django-reversion.readthedocs.io/en/stable/).